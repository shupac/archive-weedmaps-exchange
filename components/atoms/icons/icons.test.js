import React from 'react';
import { shallow } from 'enzyme';
import {
  ArrowDown,
  ArrowLeft,
  ArrowUp,
  ArrowRight,
  Bell,
  Brands,
  Chart,
  Dashboard,
  Flag,
  Help,
  KeyLock,
  Listings,
  Mail,
  Menu,
  Papers,
  Region,
  Settings,
  Spinner,
  Tag,
  Toolbox,
  WmLogoMini,
  Wm404,
  Plus,
} from './';

describe('Icons', () => {
  it('ArrowDown', () => {
    const tree = shallow(<ArrowDown />);
    expect(tree).toMatchSnapshot();
  });

  it('ArrowLeft', () => {
    const tree = shallow(<ArrowLeft />);
    expect(tree).toMatchSnapshot();
  });

  it('ArrowUp', () => {
    const tree = shallow(<ArrowUp />);
    expect(tree).toMatchSnapshot();
  });

  it('ArrowRight', () => {
    const tree = shallow(<ArrowRight />);
    expect(tree).toMatchSnapshot();
  });

  it('Bell', () => {
    const tree = shallow(<Bell />);
    expect(tree).toMatchSnapshot();
  });

  it('Brands', () => {
    const tree = shallow(<Brands />).dive();
    expect(tree).toMatchSnapshot();
  });

  it('Chart', () => {
    const tree = shallow(<Chart />).dive();
    expect(tree).toMatchSnapshot();
  });

  it('Dashboard', () => {
    const tree = shallow(<Dashboard />);
    expect(tree).toMatchSnapshot();
  });

  it('Flag', () => {
    const tree = shallow(<Flag />);
    expect(tree).toMatchSnapshot();
  });

  it('Help', () => {
    const tree = shallow(<Help />);
    expect(tree).toMatchSnapshot();
  });

  it('KeyLock', () => {
    const tree = shallow(<KeyLock />);
    expect(tree).toMatchSnapshot();
  });

  it('Listings', () => {
    const tree = shallow(<Listings />).dive();
    expect(tree).toMatchSnapshot();
  });

  it('Mail', () => {
    const tree = shallow(<Mail />);
    expect(tree).toMatchSnapshot();
  });

  it('Menu', () => {
    const tree = shallow(<Menu />);
    expect(tree).toMatchSnapshot();
  });

  it('Papers', () => {
    const tree = shallow(<Papers />).dive();
    expect(tree).toMatchSnapshot();
  });

  it('Region', () => {
    const tree = shallow(<Region />);
    expect(tree).toMatchSnapshot();
  });

  it('Settings', () => {
    const tree = shallow(<Settings />);
    expect(tree).toMatchSnapshot();
  });

  it('Tag', () => {
    const tree = shallow(<Tag />);
    expect(tree).toMatchSnapshot();
  });

  it('Toolbox', () => {
    const tree = shallow(<Toolbox />).dive();
    expect(tree).toMatchSnapshot();
  });

  it('Spinner', () => {
    const tree = shallow(<Spinner />);
    expect(tree).toMatchSnapshot();
  });

  it('WmLogoMini', () => {
    const tree = shallow(<WmLogoMini />);
    expect(tree).toMatchSnapshot();
  });

  it('Wm404', () => {
    const tree = shallow(<Wm404 />);
    expect(tree).toMatchSnapshot();
  });

  it('Plus', () => {
    const tree = shallow(<Plus />);
    expect(tree).toMatchSnapshot();
  });
});
