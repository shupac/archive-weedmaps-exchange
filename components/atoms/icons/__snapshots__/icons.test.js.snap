// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons ArrowDown 1`] = `
<Arrow
  className=""
  direction="down"
  fill="#999999"
  size={
    Object {
      "height": "16px",
      "width": "14px",
    }
  }
/>
`;

exports[`Icons ArrowLeft 1`] = `
<Arrow
  className=""
  direction="left"
  fill="#999999"
  size={
    Object {
      "height": "16px",
      "width": "14px",
    }
  }
/>
`;

exports[`Icons ArrowRight 1`] = `
<Arrow
  className=""
  direction="right"
  fill="#999999"
  size={
    Object {
      "height": "16px",
      "width": "14px",
    }
  }
/>
`;

exports[`Icons ArrowUp 1`] = `
<Arrow
  className=""
  direction="up"
  fill="#999999"
  size={
    Object {
      "height": "16px",
      "width": "14px",
    }
  }
/>
`;

exports[`Icons Bell 1`] = `
<icon-wrapperstyled__IconWrapper
  className=""
  size={
    Object {
      "height": "20px",
      "width": "16px",
    }
  }
>
  <svg
    height="20px"
    viewBox="0 0 16 20"
    width="16px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      fill="none"
      fillRule="evenodd"
      transform="translate(-1236 -20)"
    >
      <g
        fill="#999999"
        transform="translate(197)"
      >
        <path
          d="M1047 40c.14 0 .27-.01.4-.041a2.042 2.042 0 0 0 1.44-1.21c.1-.246.15-.513.15-.8h-4c.01 1.128.9 2.051 2.01 2.051zm6-11.282c0-3.149-1.64-5.785-4.5-6.482v-.698c0-.85-.67-1.538-1.5-1.538s-1.5.687-1.5 1.538v.698c-2.87.697-4.5 3.323-4.5 6.482v5.128l-2 2.051v1.026h16v-1.026l-2-2.05v-5.13z"
        />
      </g>
    </g>
  </svg>
</icon-wrapperstyled__IconWrapper>
`;

exports[`Icons Brands 1`] = `
<icon-wrapperstyled__IconWrapper
  className=""
  size={
    Object {
      "height": "14px",
      "width": "20px",
    }
  }
>
  <svg
    height="14px"
    viewBox="0 0 20 14"
    width="20px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      fill="none"
      fillRule="evenodd"
      id="Admin-Placements-Deals"
      transform="translate(-19 -127)"
    >
      <g
        id="Group-5"
      >
        <g
          id="Group-15"
          transform="translate(0 110)"
        >
          <g
            id="Icons/Manage"
            transform="translate(18 13)"
          >
            <g
              id="Icon/Dashboard"
            >
              <path
                d="M0 0h21v21H0z"
                id="Container"
              />
              <path
                d="M7.6974 11.853c.6246 0 1.4275.09 2.2755.2701-1.9629 1.0805-2.2755 2.521-2.2755 3.3313v2.1609H1.05V15.229c0-2.2509 4.4168-3.3767 6.6474-3.3767v.0007zm7.1378.9904c1.7401 0 5.2198.8557 5.2198 2.611v2.1609H9.616v-2.1609c0-1.756 3.4797-2.611 5.2199-2.611h-.0007zM7.6974 9.9623c-1.5617 0-2.8551-1.3059-2.8551-2.8812C4.8423 5.506 6.1363 4.2 7.6974 4.2c1.561 0 2.8551 1.3059 2.8551 2.8811 0 1.5753-1.294 2.8812-2.8551 2.8812zm7.1378.9457c-1.2941 0-2.3648-1.0804-2.3648-2.3863 0-1.3059 1.0707-2.431 2.3648-2.431 1.294 0 2.3647 1.1259 2.3647 2.431 0 1.3052-1.0707 2.3863-2.3647 2.3863z"
                fill="#00CDBE"
                fillRule="nonzero"
                id="Shape"
              />
            </g>
          </g>
        </g>
      </g>
    </g>
  </svg>
</icon-wrapperstyled__IconWrapper>
`;

exports[`Icons Chart 1`] = `
<icon-wrapperstyled__IconWrapper
  className=""
  size={
    Object {
      "height": "18px",
      "width": "18px",
    }
  }
>
  <svg
    height="18px"
    viewBox="0 0 18 18"
    width="18px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      transform="translate(-19, -221)"
    >
      <g>
        <g
          transform="translate(0, 206)"
        >
          <g
            transform="translate(17, 13)"
          >
            <g>
              <rect
                height="21"
                width="21"
                x="0"
                y="0"
              />
              <path
                d="M15.508 15.5074V11.651h-1.9285v3.8564h1.9286zm-3.8563 0V5.9112H9.7683v9.5962h1.8834zm-3.812 0V8.781H5.9113v6.7263h1.9286zM17.3915 2.1c1.0318 0 1.9286.8968 1.9286 1.9286v13.3628c0 1.0318-.8968 1.9286-1.9286 1.9286H4.0286c-1.0318 0-1.9286-.8968-1.9286-1.9286V4.0286C2.1 2.9968 2.9968 2.1 4.0286 2.1h13.3628z"
                fill="#00CDBE"
              />
            </g>
          </g>
        </g>
      </g>
    </g>
     
  </svg>
</icon-wrapperstyled__IconWrapper>
`;

exports[`Icons Dashboard 1`] = `
<Dashboard
  className=""
  fill="#00CDBE"
  size={
    Object {
      "height": "16px",
      "width": "14px",
    }
  }
/>
`;

exports[`Icons Flag 1`] = `
<Flag
  className=""
  fill="#00CDBE"
  size={
    Object {
      "height": "16px",
      "width": "14px",
    }
  }
/>
`;

exports[`Icons Help 1`] = `
<Help
  className=""
  fill="#00CDBE"
  size={
    Object {
      "height": "16px",
      "width": "16px",
    }
  }
/>
`;

exports[`Icons KeyLock 1`] = `
<Keylock
  className=""
  fill="#00CDBE"
  size={
    Object {
      "height": "16px",
      "width": "14px",
    }
  }
/>
`;

exports[`Icons Listings 1`] = `
<icon-wrapperstyled__IconWrapper
  className=""
  size={
    Object {
      "height": "16px",
      "width": "16px",
    }
  }
>
  <svg
    height="16px"
    viewBox="0 0 16 16"
    width="16px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      fill="none"
      fillRule="evenodd"
      transform="translate(-20 -80)"
    >
      <g
        fill="#00CDBE"
        fillRule="nonzero"
      >
        <g
          transform="translate(20 78)"
        >
          <g
            transform="translate(0 2)"
          >
            <path
              d="M14.58 7.2c-.45 0-.89-.14-1.25-.39-.03-.02-.07-.02-.12 0a2.21 2.21 0 0 1-2.51 0c-.03-.02-.07-.02-.12 0a2.21 2.21 0 0 1-2.51 0c-.02-.02-.07-.02-.12 0a2.21 2.21 0 0 1-2.51 0c-.02-.02-.07-.02-.11 0a2.21 2.21 0 0 1-2.52 0c-.02-.02-.07-.02-.11 0a2.21 2.21 0 0 1-1.67.34c-.07-.02-.12.03-.12.1v8.66c-.02.04 0 .09.07.09h1.56a.1.1 0 0 0 .09-.1v-5.77c0-.05.04-.1.09-.1h3.7c.05 0 .1.05.1.1v5.78c0 .04.04.09.09.09H15a.1.1 0 0 0 .1-.1V7.26c0-.07-.05-.12-.12-.1-.13.03-.27.05-.4.05zm-1.3 6.1H9.17a.1.1 0 0 1-.1-.09v-2.79c0-.04.05-.09.1-.09h4.11c.05 0 .1.05.1.1v2.78a.1.1 0 0 1-.1.1zm2.58-9.16c.1.18.14.39 0 .6 0 .77-.64 1.39-1.41 1.39-.53 0-.99-.28-1.24-.71-.02-.05-.11-.05-.14 0a1.43 1.43 0 0 1-2.46 0c-.03-.05-.12-.05-.14 0a1.43 1.43 0 0 1-2.47 0c-.02-.05-.11-.05-.14 0a1.43 1.43 0 0 1-2.47 0c-.02-.05-.11-.05-.13 0a1.43 1.43 0 0 1-2.47 0c-.02-.05-.12-.05-.14 0-.23.4-.7.7-1.23.7a1.4 1.4 0 0 1-1.3-1.96v-.02a.93.93 0 0 1 .1-.2l1.06-1.75.02-.04V.09c0-.04.05-.09.1-.09H14.76c.04 0 .09.05.09.1V2.3l.93 1.74c.03.02.03.04.05.06l.02.03z"
            />
          </g>
        </g>
      </g>
    </g>
  </svg>
</icon-wrapperstyled__IconWrapper>
`;

exports[`Icons Mail 1`] = `
<Mail
  className=""
  fill="#999999"
  size={
    Object {
      "height": "16px",
      "width": "24px",
    }
  }
/>
`;

exports[`Icons Menu 1`] = `
<Menu
  className=""
  fill="#999999"
  size={
    Object {
      "height": "16px",
      "width": "24px",
    }
  }
/>
`;

exports[`Icons Papers 1`] = `
<icon-wrapperstyled__IconWrapper
  className=""
  size={
    Object {
      "height": "16px",
      "width": "14px",
    }
  }
>
  <svg
    height="16px"
    viewBox="0 0 18 19"
    width="14px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      fill="none"
      fillRule="evenodd"
      id="Admin-Placements-Deals"
      transform="translate(-19 -269)"
    >
      <g
        id="Group-5"
      >
        <g
          id="Group-15"
          transform="translate(0 254)"
        >
          <g
            id="Icons/Inventory"
            transform="translate(17 14)"
          >
            <g
              id="Icon/Inventory"
              transform="translate(-1.05 1.05)"
            >
              <path
                d="M0 0h21v21H0z"
                id="Container"
              />
              <path
                d="M11.7596 13.429C8.889 11.1916 6.0285 8.9443 3.15 6.7144L11.7596 0l8.6097 6.7145c-2.8785 2.2292-5.739 4.4765-8.6097 6.7144zm0 2.4787l7.0406-5.543 1.5698 1.217-8.6096 6.7145-8.6097-6.7144 1.5698-1.217 7.0391 5.5429z"
                fill="#00CDBE"
                fillRule="nonzero"
                id="Shape"
              />
            </g>
          </g>
        </g>
      </g>
    </g>
  </svg>
</icon-wrapperstyled__IconWrapper>
`;

exports[`Icons Plus 1`] = `
<svg
  height="16px"
  viewBox="0 0 18 18"
  width="16px"
>
  <g
    fill="#DEDDDA"
  >
    <polygon
      points="0 7 0 11 18 10.92 18 7"
    />
    <rect
      height="18"
      width="4"
      x="7"
      y="0"
    />
  </g>
</svg>
`;

exports[`Icons Region 1`] = `
<Region
  className=""
  fill="#00CDBE"
  size={
    Object {
      "height": "16px",
      "width": "15px",
    }
  }
/>
`;

exports[`Icons Settings 1`] = `
<Settings
  className=""
  fill="#00CDBE"
  size={
    Object {
      "height": "15px",
      "width": "15px",
    }
  }
/>
`;

exports[`Icons Spinner 1`] = `
<svg
  height="22px"
  viewBox="0 0 32 32"
  width="22px"
>
  <circle
    cx={16}
    cy={16}
    fill="none"
    r={12}
    role="presentation"
    stroke="#9FA9BA"
    strokeDasharray={56.548667764616276}
    strokeLinecap="round"
    strokeWidth={2}
  >
    <animateTransform
      attributeName="transform"
      dur="0.9s"
      from="0 16 16"
      repeatCount="indefinite"
      to="360 16 16"
      type="rotate"
    />
  </circle>
</svg>
`;

exports[`Icons Tag 1`] = `
<Tag
  className=""
  fill="#00CDBE"
  size={
    Object {
      "height": "16px",
      "width": "14px",
    }
  }
/>
`;

exports[`Icons Toolbox 1`] = `
<icon-wrapperstyled__IconWrapper
  className=""
  size={
    Object {
      "height": "16px",
      "width": "17px",
    }
  }
>
  <svg
    height="16px"
    viewBox="0 0 17 16"
    width="17px"
    xmlns="http://www.w3.org/1999/xlink"
  >
    <defs>
      <path
        d="M6.8 11.05v-.85H.8585L.85 13.6c0 .9435.7565 1.7 1.7 1.7h11.9c.9435 0 1.7-.7565 1.7-1.7v-3.4H10.2v.85H6.8zm8.5-7.65h-3.4085V1.7l-1.7-1.7h-3.4l-1.7 1.7v1.7H1.7C.765 3.4 0 4.165 0 5.1v2.55c0 .9435.7565 1.7 1.7 1.7h5.1v-1.7h3.4v1.7h5.1c.935 0 1.7-.765 1.7-1.7V5.1c0-.935-.765-1.7-1.7-1.7zm-5.1 0H6.8V1.7h3.4v1.7z"
        id="path-toolbox1"
      />
    </defs>
    <g>
      <use
        fill="#00CDBE"
        id="Icon"
        xlinkHref="#path-toolbox1"
      />
    </g>
  </svg>
</icon-wrapperstyled__IconWrapper>
`;

exports[`Icons Wm404 1`] = `
<img
  alt="Weedmaps"
  className="wm-icon wm-404-icon"
  height="120px"
  src="/static/images/404.svg"
  width="120px"
/>
`;

exports[`Icons WmLogoMini 1`] = `
<svg
  className="wm-icon wm-logo-mini"
  height={19}
  viewBox="0 0 33 19"
  width={33}
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    fill="none"
  >
    <path
      d="M.46.893v9.35h29.645V.893H.46z"
    />
    <path
      d="M18 .373v1.232h.035c.332-.47.73-.833 1.2-1.092.465-.258 1-.387 1.605-.387.58 0 1.11.11 1.59.334.478.223.842.617 1.09 1.18.272-.4.642-.75 1.11-1.057.468-.305 1.02-.457 1.66-.457.485 0 .935.06 1.35.176.414.117.768.305 1.064.563.296.258.527.596.692 1.013.166.417.25.92.25 1.506v6.092h-2.522v-5.16c0-.304-.012-.59-.036-.86-.023-.272-.088-.506-.194-.707-.107-.2-.264-.36-.47-.48-.208-.12-.49-.18-.844-.18-.355 0-.642.07-.86.2-.22.132-.392.31-.516.53-.124.212-.207.46-.25.74-.04.27-.06.55-.06.832v5.07h-2.52v-5.1c0-.27-.007-.54-.02-.8-.01-.267-.06-.51-.15-.733-.09-.224-.237-.4-.443-.537-.2-.13-.51-.2-.91-.2-.12 0-.27.03-.47.08-.19.05-.38.15-.57.3-.18.15-.34.36-.47.637-.13.28-.19.64-.19 1.086v5.28h-2.52V.373H18zM8.752 9.476l-1.634-6.11h-.035L5.52 9.476H2.91L0 .373h2.663l1.686 6.18h.03L5.91.373h2.452l1.562 6.163h.035L11.64.373h2.592L11.34 9.476H8.75z"
      fill="#00CDBE"
      transform="translate(.46 .767)"
    />
    <path
      d="M31.72 3.01h1.204V.604h-2.41V3.01h1.205z"
    />
    <path
      d="M.92 1.12h.106c.15 0 .243-.004.28-.016.036-.01.064-.03.085-.06.03-.03.04-.062.04-.1C1.43.91 1.42.878 1.4.85 1.38.823 1.35.803 1.31.79 1.27.78 1.18.773 1.03.773H.92v.348zm-.26.727V.57h.26c.244 0 .38 0 .41.003.086.007.153.025.2.05.05.028.088.07.12.124.032.055.048.116.048.183 0 .092-.03.17-.09.236-.06.066-.145.106-.254.122.038.014.068.03.09.048.02.017.05.05.086.098.01.013.04.064.094.155l.15.25h-.32l-.107-.21c-.072-.14-.132-.23-.18-.268-.046-.036-.106-.055-.18-.055H.922v.525H.66zm.547-1.61C1.042.237.883.277.73.36.578.442.458.56.37.716c-.087.155-.13.317-.13.487 0 .17.042.33.128.482.085.153.204.273.358.358.153.086.313.13.48.13.17 0 .33-.044.484-.13.152-.085.272-.205.358-.358.086-.153.13-.313.13-.482 0-.17-.045-.332-.133-.487C1.958.56 1.838.442 1.685.36 1.532.277 1.373.236 1.207.236zm0-.237c.205 0 .404.052.594.155.19.103.34.25.45.443.11.193.17.395.17.605 0 .21-.05.41-.16.6-.1.19-.25.337-.44.444-.19.106-.39.16-.6.16-.21 0-.41-.054-.6-.16-.19-.107-.34-.255-.445-.445-.107-.19-.16-.39-.16-.6C0 .992.055.792.165.6.27.406.42.258.61.155.8.052 1 0 1.207 0z"
      fill="#00CDBE"
      transform="translate(30.513 .604)"
    />
    <path
      d="M15.39 15.578c5.448 0 8.206-2.052 8.206-2.052l1.172 1.573s-4.068 3.42-9.38 3.42-9.38-3.42-9.38-3.42l1.174-1.58s2.758 2.05 8.208 2.05"
      fill="#FFF"
    />
  </g>
</svg>
`;
